package.json

```diff
@@ -5,7 +5,7 @@
     "main": "dist/index.js",
     "homepage": "http://schul-cloud.github.io/edtrio",
     "scripts": {
-        "dev": "parcel src/index.html -d dist/dev --open",
+        "dev": "parcel src/index.html -d dist/dev --hmr-port=1235",
```

Comment:
Welchen Effekt genau hat --open in Doku steht nur:
Open in browser
Default: open disabled
Available in: serve
parcel entry.js --open

Das klinkt erstmal auch nicht schlecht. Ist aber dann vermutlich Port 80? 8000? 8080?